stdlib/pathlib.pyi

```diff
@@ -222,6 +222,7 @@ class Path(PurePath):
     else:
         def write_text(self, data: str, encoding: str | None = None, errors: str | None = None) -> int: ...
     if sys.version_info < (3, 12):
+        @deprecated("Deprecated since Python 3.10 and scheduled for removal in Python 3.12. Use hardlink_to() instead.")
```

Comment:
```suggestion
        @deprecated("Deprecated as of Python 3.10; removed in Python 3.12. Use hardlink_to() instead.")
```
nit: we know that the removal has come to pass

Reply:
thanks! I recreated your suggestion in a follow up commit